lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -251,8 +234,8 @@ public function getDateDiffExpression($date1, $date2)
      */
     public function getDateTimeTypeDeclarationSQL(array $fieldDeclaration)
     {
-        if (isset($fieldDeclaration['version']) && $fieldDeclaration['version'] == true) {
-            return "TIMESTAMP(0) WITH DEFAULT";
+        if (isset($fieldDeclaration['version']) && $fieldDeclaration['version'] === true) {
```

Comment:
This can be considered a BC break. Could be replaced with `if (!empty($fieldDeclaration['version']))`.